src/Test/Perf/util/PerfTestBase.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Roslyn.Test.Performance.Utilities
+{
+    public abstract class PerfTest : RelativeDirectory
+    {
+        private List<Tuple<int, string, object>> _metrics = new List<Tuple<int, string, object>>();
+        protected ILogger _logger;
+
+        public PerfTest(ILogger logger, [CallerFilePath] string workingFile = "") : base(workingFile) { _logger = logger; }
+
+        /// Reports a metric to be recorded in the performance monitor.
+        protected void Report(ReportKind reportKind, string description, object value)
+        {
```

Comment:
Nobody uses the Report method anymore. `_metrics` and `Report(...)` can be removed